.github/workflows/ruby.yml

```diff
@@ -22,4 +22,4 @@ jobs:
       run: |
         gem install bundler
         bundle install --jobs 4 --retry 3
-        bundle exec rake
+        bundle exec rake
```

Comment:
That little sign at the end means there wasn't an empty newline at the end of the file, which can cause some weird visual behavior sometimes in unix based systems. You can configure your text editor to add an empty line at the end for you

Reply:
Of course it didn't show if I'm making a comment about it haha

![Screen Shot 2022-10-13 at 13 19 12](https://user-images.githubusercontent.com/6660202/195663601-2647105d-7542-4a20-a304-a45bd1690e90.jpg)

Reply:
My bad, brought back ending new line with https://github.com/rubiconmd/injectable/pull/29/commits/f8e0f3d1af765e11498fdc2c990c0398c8faeffa

My [personal editorconfig states the opposite](https://github.com/bertocq/dotfiles/blob/master/tools/.editorconfig#L16) to your expectations.

But I've got no strong opinion about it, so to solve it I've added the same `.editorconfig` file that the private repos use with https://github.com/rubiconmd/injectable/pull/29/commits/93f28927c48e2a37ff905b23def0f02cc45eb005 (plus a simple explanation/reminder to configure it if needed)

Reply:
Haha, no worries, thank you for being understanding